lib/iex/lib/iex/info.ex

```diff
@@ -65,11 +65,14 @@ defimpl IEx.Info, for: Atom do
   end
 
   defp format_time({year, month, day, hour, min, sec}) do
-    "#{year}-#{month}-#{day} #{hour}:#{min}:#{sec}"
+    "#{year}-#{zeropad(month)}-#{zeropad(day)} #{zeropad(hour)}:#{zeropad(min)}:#{zeropad(sec)}"
   end
 
   defp default_or_apply(nil, _), do: "no value found"
   defp default_or_apply(data, fun), do: fun.(data)
+
+  defp zeropad(number) when number < 10, do: "0#{number}"
```

Comment:
That would be fine by me. Could copy that implementation as well for consistency. Going to bed now, hope to do it tomorrow.